api/v1/swagger.yaml

```diff
@@ -6357,10 +6357,10 @@ definitions:
           This allows a lens to render sample data in different ways depending on
           their type.
       createdBy:
+        type: string
```

Comment:
since aspect is also modified in this pull request, the type is added to the aspect response.